app/root.tsx

```diff
@@ -165,6 +174,10 @@ export function useRecaptchaSiteKey() {
   return useLoaderDataRoot()?.recaptchaSiteKey
 }
 
+export function WithCredentials() {
```

Comment:
React hooks should have names that start with `use`:

```suggestion
export function useCredentials() {
```